### ISSUE TITLE
Wrap long `page_numbering_style`

### ISSUE THREAD

**Issue by sinaatalay** [sinaatalay/RenderCV#165]
One more question is how to wrap `page_numbering_style`. By default it flows beyond the page when rendering a longer text. Looks like this part is responsible for rendering it:
```
\makeatletter
\let\ps@customFooterStyle\ps@plain % Copy the plain style to customFooterStyle
\patchcmd{\ps@customFooterStyle}{\thepage}{
    \seqsplit\color{gray}\textit{\small <<design.page_numbering_style|replace_placeholders_with_actual_values(page_numbering_style_placeholders)>>}
}{}{} % replace number by desired string
\makeatother
\pagestyle{customFooterStyle}
```

Currently I am lurking through Stack Exchange to find a way how to wrap it according to margins like any other entries. Bottom right corner of my doc:
![image](https://github.com/user-attachments/assets/023beed0-b694-4529-ac8b-1848ff27464c)

_Originally posted by @mattmaniak in https://github.com/sinaatalay/rendercv/discussions/162#discussioncomment-10477911_

**Comment by mattmaniak** [comment]
Thanks. Is it fixed in 2.0?

**Comment by mattmaniak** [comment]
> Thanks. Is it fixed in 2.0?

Bump.

**Comment by sinaatalay** [comment]
Could you please confirm it's fixed? I don't remember. It should be fixed as I closed it.

**Comment by mattmaniak** [comment]
Yes, this particular issue is fixed in 2.2 at least. Thanks for fixing. On the other hand it is still overflowing vertically. I made a separate Issue to address this.
